src/index.html

```diff
@@ -1,12 +1,333 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body body">
+    <header class="body__header header">
+      <a href="#home" class="header__logo-link">
+        <img
+          src="images/logo.png"
+          alt="logo moyo"
+          class="header__logo-img"
+        >
+      </a>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple" class="nav__link is-active">Apple</a></li>
+          <li class="nav__item"><a href="#samsung" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#smartphones" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
+            <a
+              href="#laptops&computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item"><a href="#gadgets" class="nav__link">Gadgets</a></li>
+          <li class="nav__item"><a href="#tablets" class="nav__link">Tablets</a></li>
+          <li class="nav__item"><a href="#photo" class="nav__link">Photo</a></li>
+          <li class="nav__item"><a href="#video" class="nav__link">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="body__field field">
+      <section class="field__card card" data-qa="card">
+        <div class="card__img-box">
+          <img
+            src="images/imac.jpeg"
+            alt="imac"
+            class="card__image"
+          >
+        </div>
```

Comment:
due to checklist we need to have spaces between childs

Reply:
ok